### PR TITLE
YLP-2016 Migrate BolusTooltip to dumb

### PR DESCRIPTION
**Pull request by celia-rouis**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

**Comment by celia-rouis** [comment]
Related PRs:
- data-loader: https://github.com/mdblp/data-loader/pull/35
- yourloops: https://github.com/mdblp/yourloops/pull/840

File: packages/dumb/src/components/tooltips/bolus-tooltip/bolus-tooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+      side={side || COMMON_TOOLTIP_SIDE}
+      tailHeight={COMMON_TOOLTIP_TAIL_HEIGHT}
+      tailWidth={COMMON_TOOLTIP_TAIL_WIDTH}
+      tail={DEFAULT_TOOLTIP_TAIL}
+      offset={DEFAULT_TOOLTIP_OFFSET}
+      borderWidth={DEFAULT_TOOLTIP_BORDER_WIDTH}
+      content={
+        <div className={styles.container} id="bolus-tooltip-content">
+          {isWizard && carbs &&
+            <div className={commonStyles.row} id="bolus-tooltip-line-carbs">
```
Comment (TimCanu) [inline packages/dumb/src/components/tooltips/bolus-tooltip/bolus-tooltip.tsx:135]: opt:
You could have a components taking `label`, `value` and `units` as optionnal props and display them as this div does it. Less code duplication, but a bit more logic. Do as you prefer.

**Review by TimCanu — COMMENTED** [review]
I have not tested it yet

File: packages/medical-domain/src/domains/repositories/medical/datum/wizard.service.ts [hunk 2]
```diff
@@ -50,6 +53,12 @@ const normalize = (rawData: Record<string, unknown>, opts: MedicalDataOptions):
       net: recommended.net as number
     }
   }
+  if (rawData.inputMeal) {
+    const inputMeal = rawData?.inputMeal as Record<string, unknown>
```
[the remark this answers is not in the document]
Comment (celia-rouis) [reply packages/medical-domain/src/domains/repositories/medical/datum/wizard.service.ts:57]: I don't think it prevents anything tbh, it was just a typing issue 😄

File: packages/medical-domain/src/domains/repositories/medical/datum/wizard.service.ts [hunk 3]
```diff
@@ -30,26 +30,35 @@ import { DatumProcessor } from '../../../models/medical/datum.model'
...
     wizard.recommended = {
       carb: recommended.carb as number,
       correction: recommended.correction as number,
       net: recommended.net as number
     }
   }
+  if (rawData?.inputMeal) {
+    const inputMeal = rawData.inputMeal as Record<string, unknown>
+    wizard.inputMeal = {
+      fat: inputMeal?.fat as WizardInputMealFat
```
Comment (TimCanu) [inline packages/medical-domain/src/domains/repositories/medical/datum/wizard.service.ts:59]: Wait, is `inputMeal` still nullable?

**Pull request merged**